crates/oxc_linter/src/rules/react/iframe_missing_sandbox.rs

```diff
@@ -0,0 +1,233 @@
+    ///
+    /// Examples of **incorrect** code for this rule:
+    /// ```jsx
+    /// <iframe/>;
+    /// <iframe sandbox="invalid-value" />;
+    /// ```
+    ///
+    /// Examples of **correct** code for this rule:
+    /// ```jsx
+    /// <iframe sandbox="" />;
+    /// <iframe sandbox="allow-origin" />;
+    /// ```
+    IframeMissingSandbox,
+    style,
+    suggestion
+);
+
+impl Rule for IframeMissingSandbox {
+    fn run<'a>(&self, node: &AstNode<'a>, ctx: &LintContext<'a>) {
+        match node.kind() {
+            AstKind::JSXOpeningElement(jsx_el) => {
+                let JSXElementName::Identifier(identifier) = &jsx_el.name else {
+                    return;
+                };
+
+                let name = identifier.name.as_str();
+                if name != "iframe" {
+                    return;
+                }
+
+                has_jsx_prop_ignore_case(jsx_el, "sandbox").map_or_else(
+                    || {
+                        ctx.diagnostic(missing_sandbox_prop(identifier.span));
+                    },
+                    |sandbox_prop| {
+                        self.validate_sandbox_attribute(sandbox_prop, ctx);
+                    },
+                );
+            }
+            AstKind::CallExpression(call_expr) => {
+                if is_create_element_call(call_expr) {
+                    let Some(Argument::StringLiteral(str)) = call_expr.arguments.first() else {
+                        return;
+                    };
+
+                    if str.value.as_str() != "iframe" {
+                        return;
+                    }
+
+                    if let Some(Argument::ObjectExpression(obj_expr)) = call_expr.arguments.get(1) {
+                        obj_expr
+                            .properties
+                            .iter()
+                            .find_map(|prop| {
+                                if let ObjectPropertyKind::ObjectProperty(prop) = prop {
+                                    if prop.key.is_specific_static_name("sandbox") {
+                                        return Some(prop);
+                                    }
+                                }
+
+                                None
+                            })
+                            .map_or_else(
+                                || {
+                                    ctx.diagnostic(missing_sandbox_prop(obj_expr.span));
+                                },
+                                |sandbox_prop| {
+                                    self.validate_sandbox_property(sandbox_prop, ctx);
+                                },
+                            );
+                    } else {
+                        ctx.diagnostic(missing_sandbox_prop(call_expr.span));
+                    }
+                }
+            }
+            _ => {}
+        }
+    }
+}
+
+impl IframeMissingSandbox {
+    #[allow(clippy::unused_self)]
+    fn get_allowed_values(&self) -> Vec<&str> {
```

Comment:
Not sure how I can bypass this

Reply:
the `#[allow(...` ? the `fn` can just be moved out of the impl ?